packages/demo-react-native/App/Config/ReactotronConfig.js

```diff
@@ -28,4 +28,7 @@ if (__DEV__) {
   Reactotron.clear()
 }
 
+const testUnreg = Reactotron.onCustomCommand('test', () => { console.log('I WORK!') })
+Reactotron.onCustomCommand('test2', () => { testUnreg() })
```

Comment:
If this is the direction we go I would think we would want to setup a better demo but this serves its purpose - which is to simply show off that I can write code that works all of then time...60% of the time.

Reply:
haha